lib/ansible/modules/service_facts.py

```diff
@@ -70,7 +70,7 @@
         state:
           description:
           - State of the service.
-          - 'This commonly includes (but is not limited to) the following: C(failed), C(running), C(stopped) or C(unknown).'
+          - 'This commonly includes (but is not limited to) the following: C(running), C(failed), C(dead), C(exited), C(stopped) or C(unknown).'
```

Comment:
exited seems to be reasons for failed/stopped, not really different states, unsure also about dead/failed

Reply:
its service_facts, not systemd_facts, I don't really want to pollute with states that are  nonsensical to other systems

Reply:
I get it, was just explaining my reasoning. I will revert it to how it was:
```yaml
  - 'This commonly includes (but is not limited to) the following: C(failed), C(running), C(stopped) or C(unknown).' 
```

Reply:
done.